content/workflows/csp/aws/service/ec2/resources/route-table/policy-context/vpc-route-table-not-traffic-restricted.yaml

```
@@ -0,0 +1,23 @@
+schema: 1.0
+type: policy-context
+id: vpc-route-table-not-traffic-restricted
```

```suggestion
id: vpc-route-table-is-overly-permissive
```

Please rename file as well